lib/Jsrt/JsrtDebuggerObject.cpp

```diff
@@ -826,7 +814,7 @@ Js::JavascriptArray * JsrtDebugStackFrames::StackFrames(Js::ScriptContext * scri
         }
         else
         {
-            this->framesDictionary->Clear();
+            this->ClearFrameDictionary();
```

Comment:
No, its allocated just once (the first time). deleted and removed in JsrtDebugStackFrames destructor. 

---
In reply to: [77228263](https://github.com/Microsoft/ChakraCore/pull/1521#discussion_r77228263) [](ancestors = 77228263)